Update workshop description, and add featured workshop

mirisuzanne opened the pull request:
![](https://source.unsplash.com/featured/?cute,animal)

SondraE reviewed (approved):
@mirisuzanne 
I like what you've done here with the featured workshop and adding the word "workshop" to the subtitles of the items in the list. (This makes me realizes that our subtitle upper/lower cases are all over the place for talks and workshops. Might be nice to standardize, but not something that needs to hold up this PR.)

The other thing I felt could be useful is a little blurb to go along with the "schedule a workshop" CTA just making it really clear that people can hire us to give this workshop in-house in addition to the scheduled events.

mirisuzanne commented:
> This makes me realizes that our subtitle upper/lower cases are all over the place for talks and workshops. Might be nice to standardize, but not something that needs to hold up this PR.

@SondraE If you have a preference, I can at least get these all consistent.

> The other thing I felt could be useful is a little blurb to go along with the "schedule a workshop" CTA just making it really clear that people can hire us to give this workshop in-house in addition to the scheduled events.

Is this too much? `Schedule a workshop for your company or conference »`

stacyk commented on content/talks.njk:
```diff
@@ -1,25 +1,21 @@
-  src: talks/miriam.jpg
-  alt: Miriam Suzanne speaking at a conference
-  position: left center
+sub: Bring us to your event, or browse past talks
```
Is the comma needed here? It wasn't there before and we only changed 1 word.

stacyk commented on content/workshops/django-end-to-end-testing.md:
```diff
@@ -1,5 +1,6 @@
 ---
 title: Effective End-to-End Testing for Django
+sub: A Workshop
```
Is this missing the rest of the line? 
![Screenshot 2023-10-23 at 3 27 54 PM](https://github.com/oddbird/oddleventy/assets/1581694/5cb352f1-9749-4481-9572-d2ef2f59d429)

stacyk commented on content/talks.njk:
```diff
@@ -30,14 +26,38 @@ summary: |
+      {% endif %}
+    </div>
+  </article>
+{% endif %}
+
 {{ talks.section(
   title='Upcoming Events',
   events=collections.all | getPublic | getEvents | getFuture,
   all=false,
   by_bird=collections
 ) }}
```
@mirisuzanne Did we always show upcoming events like this?  

My suggestions would be to have the name of the event on top, a description (or tickets on sale now) and the presenter's name, then the date/s like the other talk cards. 

Are we showing the presenters sketch here because we don't have an image, or did we always do this?

![Screenshot 2023-10-23 at 3 28 28 PM](https://github.com/oddbird/oddleventy/assets/1581694/af318ec9-a195-47dd-a9bb-ec089e9f5dfa)

mirisuzanne replied:
@stacyk  I didn't make any changes here, this is how we've been displaying future events - grouped by presenter. I think this should be filed as a new issue if we want to revisit that design.

stacyk replied:
@SondraE Would this be one of those shapeshifters?

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+    adr: Online
+    url: https://smashingconf.com/TBD
+    note: >
+      Tickets [on sale now](https://smashingconf.com/TBD)!
```
This link will have to be updated (or removed) before we publish this branch

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
-  to develop resilient and delightful experiences
-  that hold up across browsers, languages, and device interfaces.
+  New CSS features are shipping at an unprecedented rate --
+  cascade layers, container queries, the ':has()' selector,
```
should :has() but in the tick marks? like `:has()`?

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+or stick with the tools that I know?
+
+If you ever feel lost in the CSS weeds,
+wrestling with  the cascade,
```
```suggestion
wrestling with the cascade,
```

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+- The entire cascade, why it exists,
+  and what all is involved (much more than specificity!)
+- Managing style conflicts with cascade layers, scope, and nesting.
+- Modern selectors like :is(), :where(), and :has().
```
same here with the tick marks?

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+- The layout flow with intrinsic and extrinsic sizing.
+- Size comparison functions like min(), max(), and clamp().
+- When to use flexbox vs grid for layout and alignment.
+- Fast and dynamic layouts using subgrid and container queries.
```
I feel like most of these list items aren't actually sentences, so I wonder if including the `.` at the end of each line is necessary?

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+
+#### First, Do No Harm
+
+CSS is fundamentally different from other languages or design tools, built around a radical vision for contextual style and user-control. We’ll dig into the practical implications of that vision, and how the ‘grain’ of the language can guide us to more performant and resilient styles.
+
```
lint the line length on these sections?

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+CSS custom properties (aka “variables”) expose the internals of CSS value resolution and error recovery. What does it even mean for properties to become ‘invalid at computed value time’? And how can we use these CSS internals to our advantage, developing more robust and dynamic style systems?
+
+#### CSS Variables in Practice
+Practical use-cases, and interactive exercises related to CSS variables and functions…
```
Is the ellipsis a placeholder for content not yet written?

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+
+#### Flex & Alignment in Practice
+
+Practical use-cases, and interactive exercises related to distributing space…
```
Same question re: ellipsis

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+
+#### Grids & Containers in Practice
+
+Practical use-cases, and interactive exercises related to defining layouts…
```
Same question re: ellipsis

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+
+#### Defining Structure
+
+Sometimes we also need to impose external structure to create consistent and reliable layouts, even with unpredictable content. This is a full session dedicated to Grids & Sub-grids, the multiple ‘stages of squishiness’, and container queries.
```
does grid and subgrid need to be capitalized here (as we aren't capitalizing a number of other techniques). Also, does subgrid need the hyphen?

stacyk commented on content/workshops/cascading-styles-deep.md:
```diff
@@ -17,71 +21,120 @@ tags:
+
+#### Cascade Aligned Programming
+
+There are many conventions for organizing CSS, but the best systems all share underlying principles based on CSS itself. What would it mean to use The Cascade as our guide for writing maintainable CSS?
```
does "The Cascade" need to be capitalized here? Maybe we should use <em> if we want to highlight this?

stacyk reviewed (commented):
@mirisuzanne Left a few questions here, and sometimes I didn't leave the same type of comments if it happens more than once (like line length or adding tick marks to some items). I also don't fully understand the status of some of these things so maybe there are great reasons for things like the ellipsis :)

SondraE commented:
**Subtitles**
It looks like we typically capitalize the first letter (both here and on posts) unless there is something like a proper noun or an acronym, so I recommend that option.

**CTA**
That's on the long side for a CTA. I personally like to keep them as short as possible so they are extremely easy to skim, but if there's no obvious place to add more information without a bigger redesign, then we can go with that.

mirisuzanne commented:
@SondraE @stacyk I think I've addressed all your comments here, besides the placeholder link to Smashing. I think we can wait to merge until they do? Let me know if you see any other issues.

stacyk reviewed (approved):
Changes look good to me!
